samwise.js

```diff
@@ -2,34 +2,34 @@
  * Autoplay plugin - Automatically advance slideshow after N seconds
```

Comment:
SCRIPT-Consider extracting this repeated logic into a separate function to avoid duplication.